redirections.yml

```diff
@@ -605,6 +605,9 @@
   -
     old: "/databases/mysql/workbench"
     new: "/databases/mysql/accessing"
+  -
+    old: "/databases/postgresql/managing#upgrading"
+    new: "/databases/postgresql/upgrading"
```

Comment:
@Frzk did you checked if this kind of redirection works? I don't think it does and from what is being deployed in prod, it does not seem to work (https://doc.scalingo.com/databases/postgresql/managing#upgrading).

Reply:
Indeed, while it seems supported by the protocol, it's poorly implemented and doesn't work with our services. I removed it in #2883

Reply:
Removed in #2885